boards/amd/versalnet_rpu/Kconfig.defconfig

```diff
@@ -0,0 +1,10 @@
+#
+# Copyright (c) 2025 Advanced Micro Devices, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+CONFIG_ARM_MPU=y
+CONFIG_CACHE_MANAGEMENT=y
+CONFIG_DCACHE=y
+CONFIG_ICACHE=y
```

Comment:
These items are already provided for any Cortex-R52 CPU in arch/arm/core/cortex_a_r/Kconfig.

Reply:
okay will remove